fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/converters/PropertyConverter.java

```diff
@@ -128,7 +128,10 @@ public static String getPropertyNameFromPredicate(final NamespaceRegistry namesp
         }
 
         final String rdfLocalname = predicate.getLocalName();
-
+        if (rdfLocalname.isEmpty()) {
+          LOGGER.info("predicate local name is null");
```

Comment:
This is inaccurate. The local name is _empty_, not _null_. How about "Predicate local name is empty!"